docs/source/ko/_toctree.yml

```
@@ -137,6 +137,8 @@
     title: 문제 해결
   - local: gguf
     title: GGUF 파일들과의 상호 운용성
+  - local: modular_transformers
+    title: `transformers`에서의 모듈성
```

Ah sorry, I think we just can't have any other characters in the title here! 😅 

```suggestion
    title: transformers에서의 모듈성
```